### PR TITLE
[VEN-2497][VEN-2505]:  IL core pool configuration on Arbitrum

### PR DESCRIPTION
**Pull request by Debugger022**
## Description

Resolves VEN-2497 and VEN-2505
This **PR** includes VIP to configure the IL(Core pool) on **Arbitrum One** and **Arbitrum Sepolia**.

### CODE REVIEW

**Review by chechu — APPROVED** [review]

**Comment by Debugger022** [comment]
Multisig tx Arbitrum Sepolia: https://sepolia.arbiscan.io/tx/0x30a58d1906b7290425c3d84b329b48944f5a7b9fdabe8e83be1356dfb9226b6c

**Comment by Debugger022** [comment]
Multisig tx for new IL addresses configuration: https://sepolia.arbiscan.io/tx/0xc4547d8f45a445ddf4fc3446758a2907d15cb9c9f3d2e53c4c9fcd0025c888e1

File: multisig/simulations/arbitrumone/vip-002/index.ts [hunk 1]
```diff
@@ -0,0 +1,403 @@
...
+
+const riskParameters: { [key in VTokenSymbol]: RiskParameters } = {
+  // Core Pool
+  vWBTC_Core: {
+    borrowCap: "500",
+    supplyCap: "900",
+    collateralFactor: "0.75",
+    liquidationThreshold: "0.80",
+    reserveFactor: "0.2",
+    initialSupply: "0.3",
```
[the remark this answers is not in the document]
Comment (Debugger022) [reply multisig/simulations/arbitrumone/vip-002/index.ts]: Done https://github.com/VenusProtocol/vips/pull/248/commits/bc03de5b21b042cabe1c0357fb8a62081e68b386

File: multisig/proposals/arbitrumone/vip-002/index.ts [hunk 2]
```diff
@@ -0,0 +1,341 @@
...
+    },
+    {
+      target: arbitrumone.POOL_REGISTRY,
+      signature: "addPool(string,address,uint256,uint256,uint256)",
+      params: ["Core", COMPTROLLER_CORE, parseUnits("0.5", 18), parseUnits("1.1", 18), parseUnits("100", 18)],
+    },
+    {
+      target: arbitrumone.VTREASURY,
+      signature: "withdrawTreasuryToken(address,uint256,address)",
+      params: [WBTC, parseUnits("0.3", 8), arbitrumone.NORMAL_TIMELOCK],
```
[the remark this answers is not in the document]
Comment (Debugger022) [reply multisig/proposals/arbitrumone/vip-002/index.ts]: Done https://github.com/VenusProtocol/vips/pull/248/commits/bc03de5b21b042cabe1c0357fb8a62081e68b386

File: multisig/proposals/arbitrumone/vip-002/index.ts [hunk 3]
```diff
@@ -0,0 +1,341 @@
...
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "addPool(string,address,uint256,uint256,uint256)", arbitrumone.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setPoolName(address,string)", arbitrumone.NORMAL_TIMELOCK],
```
[the remark this answers is not in the document]
Comment (Debugger022) [reply multisig/proposals/arbitrumone/vip-002/index.ts]: Done https://github.com/VenusProtocol/vips/pull/248/commits/bc03de5b21b042cabe1c0357fb8a62081e68b386

File: multisig/proposals/arbitrumone/vip-002/index.ts [hunk 4]
```diff
@@ -0,0 +1,341 @@
...
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "updatePoolMetadata(address,VenusPoolMetaData)", arbitrumone.NORMAL_TIMELOCK],
```
[the remark this answers is not in the document]
Comment (Debugger022) [reply multisig/proposals/arbitrumone/vip-002/index.ts]: Done https://github.com/VenusProtocol/vips/pull/248/commits/bc03de5b21b042cabe1c0357fb8a62081e68b386

File: multisig/proposals/arbitrumone/vip-002/index.ts [hunk 5]
```diff
@@ -0,0 +1,341 @@
...
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setActionsPaused(address[],uint256[],bool)", arbitrumone.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "addPool(string,address,uint256,uint256,uint256)", arbitrumone.NORMAL_TIMELOCK],
```
[the remark this answers is not in the document]
Comment (Debugger022) [reply multisig/proposals/arbitrumone/vip-002/index.ts]: Done https://github.com/VenusProtocol/vips/pull/248/commits/bc03de5b21b042cabe1c0357fb8a62081e68b386

File: multisig/proposals/arbitrumone/vip-002/index.ts [hunk 6]
```diff
@@ -0,0 +1,341 @@
...
+
+export const VWBTC_CORE = "0xaDa57840B372D4c28623E87FC175dE8490792811";
+export const VWETH_CORE = "0x68a34332983f4Bf866768DD6D6E638b02eF5e1f0";
+export const VUSDC_CORE = "0x7D8609f8da70fF9027E9bc5229Af4F6727662707";
+export const VUSDT_CORE = "0xB9F9117d4200dC296F9AcD1e8bE1937df834a2fD";
+export const VARB_CORE = "0xAeB0FEd69354f34831fe1D16475D9A83ddaCaDA6";
+
+// IL configuration
+const vip002 = () => {
+  return makeProposal([
```
[the remark this answers is not in the document]
Comment (Debugger022) [reply multisig/proposals/arbitrumone/vip-002/index.ts]: Done https://github.com/VenusProtocol/vips/pull/248/commits/bc03de5b21b042cabe1c0357fb8a62081e68b386

**Review by chechu — APPROVED** [review]

**Comment by chechu** [comment]
https://app.venus.io/#/governance/proposal/320?chainId=56

**Pull request merged**